e2-studio-star-rx72n-firmware/libs/rx_hal/src/gpio.c

```
@@ -566,6 +568,7 @@ rx_err_t gpio_write_high(const rx_port_pin_t pin)
 
   /* Validate parameters */
   const rx_err_t err = internal_validate_port_pin(port, pin_num);
```

Clang (14.0.6)

[warning] variable 'err' is not initialized

(cppcoreguidelines-init-variables)